packages/binding/src/types/binding-info.ts

```diff
@@ -1,39 +1,50 @@
 import { Context } from "@ui5-language-assistant/context";
 import { TextDocumentPositionParams } from "vscode-languageserver-protocol";
 
-export type TypeKind = "string" | "boolean" | "object";
-export type BindingInfoName =
-  | "path"
-  | "value"
-  | "model"
-  | "suspended"
-  | "formatter"
-  | "useRawValues"
-  | "useInternalValues"
-  | "type"
-  | "targetType"
-  | "constraints"
-  | "mode"
-  | "parameters"
-  | "events"
-  | "parts"
-  | "formatOptions";
+export enum TypeKind {
+  "string" = "string",
+  "boolean" = "boolean",
+  "object" = "object",
+}
+export enum BindingInfoName {
+  "path" = "path",
+  "value" = "value",
+  "model" = "model",
+  "suspended" = "suspended",
+  "formatter" = "formatter",
+  "useRawValues" = "useRawValues",
+  "useInternalValues" = "useInternalValues",
+  "type" = "type",
+  "targetType" = "targetType",
+  "constraints" = "constraints",
+  "mode" = "mode",
+  "parameters" = "parameters",
+  "events" = "events",
+  "parts" = "parts",
+  "formatOptions" = "formatOptions",
+}
 export interface Dependents {
   name: BindingInfoName;
   type: PropertyType[];
 }
 export interface PropertyType {
-  kind: TypeKind;
+  kind: string;
+  default?: {
```

Comment:
`default` seems a bit misleading here. It's a list of "all" possible values, not the one that would be used in case there is no other value.